content/pages/sprint/2023/02-taipei.rst

```diff
@@ -69,6 +69,24 @@ https://hackmd.io/@sciwork/BJkd6Blts
 
   <iframe width="100%" height="500" src="https://hackmd.io/@sciwork/BJkd6Blts" frameborder="0"></iframe>
 
+Event Album
```

Comment:
These are the highlights of the event.  I think it will be great for them to be shown in the beginning of the page.